src/inc/MSIXResource.hpp.cmakein

```diff
@@ -0,0 +1,56 @@
+//
+//  Copyright (C) 2019 Microsoft.  All rights reserved.
+//  See LICENSE file in the project root for full license information.
+//
+//  This file is generated by CMake. Do not edit.

```

Comment:
This isn't quite true any more.

Reply:
This is still true for the header. CMake takes MSIXResource.hpp.cmakein and creates MSIXResource.hpp 

---
In reply to: [274102137](https://github.com/Microsoft/msix-packaging/pull/82#discussion_r274102137) [](ancestors = 274102137)

Reply:
It makes the hpp, but the cmakein file (this one) isn't generated and it does need to be modified potentially.

---
In reply to: [274121009](https://github.com/Microsoft/msix-packaging/pull/82#discussion_r274121009) [](ancestors = 274121009,274102137)

Reply:
Ok, modified to "MSIXResource.hpp is generated by CMake. Do not edit"

---
In reply to: [274121740](https://github.com/Microsoft/msix-packaging/pull/82#discussion_r274121740) [](ancestors = 274121740,274121009,274102137)